client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -29,7 +28,6 @@ import {
 import {
 	getAuthorizationData,
 	getAuthorizationRemoteSite,
-	getSSOSessions,
```

Comment:
We can nix this from `client/state/jetpack-connect` altogether. It looks like we are not using it anywhere except for in unit tests.